styles/components/_modal.scss

```diff
@@ -39,4 +39,45 @@
   }
   &__button {
   }
+
+  &__win-heading {
```

Comment:
consider modal__heading--win (if modal__heading would be basic for all modals, but this one could have its perks in here)

Reply:
removed this class and content moved to modal__heading